test/helpers.js

```diff
@@ -345,3 +345,11 @@ export const increaseTime = async function(duration) {
     });
   });
 };
+
+export const concatBytes = function (bytes1, bytes2) {
+  return bytes1 + (bytes2.slice(2));
```

Comment:
Does this assume that the bytes does not include the bytes header (which include the bytes length) as in solidity type bytes ?

Reply:
Not sure I understand the question... This simply assumes the bytes format which starts like this: `"0x..."`.